docker/start-local-environment.sh

```diff
@@ -4,3 +4,4 @@ set -e
 
 docker-compose build
 docker-compose up
+docker-compose down
```

Comment:
@mattd-kainos thoughts on running down first? SO we can avoid the scenario of: start -> old images -> down -> start -> new images

to

down -> start -> new images

Reply:
I don't use this script at all and just use docker-compose commands directly. It could just be removed.

I don't think you want down as the last command though.